props/saber_fett263_buttons.h

```diff
@@ -3026,18 +3032,15 @@ SaberFett263Buttons() : PropBase() {}
       default:
         break;
       case MENU_PRESET:
-        if (direction > 0) {
-          if (SaberBase::IsOn()) {
+        if (SaberBase::IsOn()) {
+          if (direction > 0) {
             next_preset_fast();
           } else {
-            SelectPreset(current_preset_.preset_num + direction);
-          }
-        } else {
-          if (SaberBase::IsOn()) {
             previous_preset_fast();
-          } else {
-            SelectPreset(current_preset_.preset_num + direction);
           }
+        } else {
+          int preset = (current_preset_.preset_num + direction) % num_presets_;
```

Comment:
Why not just call GetNumberOfPresets() here?

Reply:
Since it's a static number I figured it was better to just get the value once when entering the mode instead of getting the same value on each turn.

Reply:
There is also a drawback to getting it once: It makes the code slightly more complicated. More complicated code is only worth it if you get something for it.  My guess is that the user will never notice the difference if you just put the call here.

Reply:
OK, consolidated to the MenuDial() case